tamr_client/categorization/project.py

```diff
@@ -21,6 +23,39 @@ def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
     )
 
 
+def create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    description: str = None,
+    external_id: str = None,
+    unified_dataset_name: str = None,
```

Comment:
Optional values should be typed with `Optional`:

```suggestion
    description: Optional[str] = None,
    external_id: Optional[str] = None,
    unified_dataset_name: Optional[str] = None,
```

If you do this^, you should omit the `(Optional)` text in the docstring description since our tooling will automatically mark a field as optional in the docs if its typed as an `Optional`.

Reply:
done